### ISSUE TITLE
Nice to have: ability to unpublish pages/files

### ISSUE THREAD

**Issue by mxmclaudel** [silverstripe/silverstripe-campaign-admin#151]
At the moment, the campaign module only allows pages and files to be published. This works well for new and updated pages and files. It does not allow archiving obsolete pages and files.

ChangeSetItem does seem to already support it:

`ChangeSetItem::CHANGE_DELETED`

**Comment by clarkepaul** [comment]
Agree, Campaigns should have access to the standard set of actions that you'd imagine with the likes of page bulk actions, so that would include Unpublish, Archive etc. 

We actually have some upcoming plans for campaigns with some additional actions/features so without promising anything, this is something we are considering with upcoming work.

**Comment by clarkepaul** [comment]
Just a few notes in case this makes its way up the line. 

- Campaign publish button should change to something like "Launch campaign" rather than using the word publish as the terminology/understanding conflicts with "unpublish".
- Allow entire campaigns to be unpublished/archived e.g. unpublishing all staff profile pictures. Effectively making it more like a bulk actions tool.
- Allow items to block the selected campaign action from happening 
- Applying alternate rules to individual items in a campaign (applies to only that campaign):
  - Unpublish at launch
  - Archive at launch
  - Block publishing of this item at launch e.g. publish homepage page but refrain from publishing the banner block
